drf/accounts/views.py

```diff
@@ -1,8 +1,16 @@
-from django.shortcuts import render
-from django.http import HttpResponse
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
 
+"""
+View for testing auth
+"""
 
-def index(request):
-    return HttpResponse("Hello, world")
 
-# Create your views here.
+@api_view(['GET'])
+# @permission_classes((IsAuthenticated, ))
+def example_view(request, format=None):
```

Comment:
Please use generics (Class based views - not functions)

Reply:
Ok, I just copied example from documentation.